llvm/lib/Target/DirectX/DXILFinalizeLinkage.cpp

```diff
@@ -19,20 +19,20 @@
 using namespace llvm;
 
 static bool finalizeLinkage(Module &M) {
-  SmallPtrSet<Function *, 8> EntriesAndExports;
+  SmallPtrSet<Function *, 8> Funcs;
 
   // Find all entry points and export functions

```

Comment:
This comment may be better updated to something like:
Collect all non-entry point and non-exported functions to possibly set to internal linkage later.

Reply:
Updated